stable_baselines/common/vec_env/dummy_vec_env.py

```diff
@@ -48,6 +48,12 @@ def step_wait(self):
         return (self._obs_from_buf(), np.copy(self.buf_rews), np.copy(self.buf_dones),
                 self.buf_infos.copy())
 
+    def seed(self, seed=None):
+        seeds = list()
+        for idx, env in enumerate(self.envs):
+            seeds.append(env.seed(seed + idx))
```

Comment:
I 'm not sure if `env.seed()` return the seed by default, I think it returns None

Reply:
From the OpenAI Gym documentation it says:

> Returns:
            list<bigint>: Returns the list of seeds used in this env's random
              number generators. The first value in the list should be the
              "main" seed, or the value which a reproducer should pass to
              'seed'. Often, the main seed equals the provided 'seed', but
              this won't be true if seed=None, for example.

But you are somehow right, as there is a default implementation in gym.Env and it just returns None. So should i still change it to None, or is it possible to declare None or List[int] as return type?

Reply:
`Optional[List[int]]` (return maybe a list maybe None) or `List[Union[None, int]]` (returns a list of either None or int) ?

Reply:
I think i am going with the second option here, thanks.